.github/workflows/pr-release.yml

```diff
@@ -164,13 +164,14 @@ jobs:
               # Append new result to the existing comment or post a new comment
               # It's essential we use the MATHLIB4_BOT token here, so that Mathlib CI can subsequently edit the comment.
               if [ -z "$existing_comment_id" ]; then
+                INTRO="Mathlib CI status ([docs](https://leanprover-community.github.io/contribute/tags_and_branches.html)):"
                 # Post new comment with a bullet point
                 echo "Posting as new comment at leanprover/lean4/issues/${{ steps.workflow-info.outputs.pullRequestNumber }}/comments"
                 curl -L -s \
                   -X POST \
                   -H "Authorization: token ${{ secrets.MATHLIB4_BOT }}" \
                   -H "Accept: application/vnd.github.v3+json" \
-                  -d "$(jq --null-input --arg val "$MESSAGE" '{"body": $val}')" \
+                  -d "$(jq --null-input --arg intro "$INTRO" --arg val "$MESSAGE" '{"body": "$intro + "\n" + $val}')" \
```

Comment:
This ought to have a corresponding change in https://github.com/leanprover-community/mathlib4/blob/f880f9c4d36144973951f8255fdea629c02feeac/scripts/lean-pr-testing-comments.sh#L102; will do that when and if we merge this.

Reply:
https://github.com/leanprover-community/mathlib4/pull/9585